src/helpers/http_client.py

```diff
@@ -0,0 +1,132 @@
+import requests
+from src.helpers.counters import execution_time_delta
+
+
+class SessionHttp(requests.Session):
```

Comment:
It's enough to have one HTTP client, leave only "ClientHttp" and don't need to inherit from requests.Session